src/components/Modal/Modal.tsx

```diff
@@ -29,12 +56,24 @@ export const Modal = (props) => {
     const { className: contentEnteredClass } = elem('content--entered', props);
     const { className: contentExitedClass } = elem('content--exited', props);
```

Comment:
Ah, I believe you copypasted `elem('body--open', props);` from here, bu these are examples of not correct usage of `elem` function.

Should be:
```ts
const { className: contentEnteredClass } = elem('content', { ...props, entered: true });
const { className: contentExitedClass } = elem('content', { ...props, exited: true });
```

Reply:
Because the way this is used in React Modal, setting it in this way makes final DOM having repeated classes, that looks messy. So I kept it as it is now. Alternatively we could rename the enter/exit classes in the scss files so that they don't use the usual modifier patterns to avoid internal confusion, e.g. `&__enteredContent` instead of `&__content--entered`.

Reply:
Ah, I see. Now I remember there was a task to cover this case in bem. Ok, let's keep it this way then so far.